xml/System.Numerics/BigInteger.xml

```diff
@@ -8139,6 +8154,7 @@ This operation performs an unsigned (otherwise known as a logical) right shift o
 
           ]]></format>
         </remarks>
+        <inheritdoc cref="M:System.Numerics.IShiftOperators`3.op_UnsignedRightShift(`0,`1)" />
```

Comment:
The tool imported this one correctly:
https://github.com/dotnet/runtime/blob/38ba41ef6f29cb7a3ca50ea93ba47abb785e2654/src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs#L5208

I don't know why mdoc is complaining, the DocID of the cref is correct:
https://github.com/dotnet/dotnet-api-docs/blob/21a3a987a75688a0a9e5b5e4751751b8402a3105/xml/System.Numerics/IShiftOperators%603.xml#L95

I'll try by removing the cref, see if that makes the suggestion go away:

```suggestion
        <inheritdoc />
```